src/main/java/core/basesyntax/SalaryInfo.java

```diff
@@ -1,7 +1,44 @@
 package core.basesyntax;
 
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+
 public class SalaryInfo {
+    private final DateTimeFormatter formatter = DateTimeFormatter.ofPattern("d.MM.yyyy");
+    private final int startPositionForSubstringDate = 0;
+    private final int endPositionForSubstringDate = 10;
```

Comment:
Magic numbers should be avoided. The positions used for substrings should be constants with informative names to explain their purpose.